guides-and-standards/guide-choosing-software-dependencies.md

```diff
@@ -0,0 +1,59 @@
+
+### Important — think carefully before proceeding with the tool if these criteria are not met
+
+- **The software code is on GitHub or another popular developer platform like GitLab or BitBucket.** Using these platforms signals that the tool is open source and likely versioned. Sometimes you'll find code on SourceForge. This is ok and potentially workable, but not the best. Sometimes you'll find code in the supplement of a paper. This is not awesome and probably signals that the tool will be hard to install or use but you can still try if you really want to.
+- **The tool is easily installable.** To be able to use a tool, we need to be able to install it. Below I’ve included somewhat arbitrary gradations of what makes something easy to install. You can check these things by eye to quickly see if you think a tool will be installable, but often the best way to determine this is to install and use the tool.
+  - Very easy: cross-platform installation as a conda package
+  - Easy: installable from another package repository with another package manager like pip or poetry (python) or `install_packages()` (R)
+  - Somewhat easy: Installable from GitHub with a package manager
+  - Somewhat easy: Has a dockerfile and an available Docker container
+  - Somewhat difficult: has an environment yaml file or requirements.txt file that lists versioned dependencies and has installation instructions that involve cloning the repository
+  - Difficult: git clone & compile from source
+  - Difficult: No installation instructions
+
+### Nice — but not necessary if you’re willing to do the leg work to make sure the tool works
+
+- **The tool has tests.** Tests ensure the accuracy and quality of a software tool. See [this AUG](https://training.arcadiascience.com/arcadia-users-group/20231104-testing-concepts/lesson/) for more about software tests.
+  - Hint: tests will often be a `test/` or `tests` folder, either in the root of the repository or
+- **The software is maintained.**
+  - Hint: there is a difference in the timestamp between the first and last commit and the last commit occurred recently (~months)
+  - Hint: there are responses to issues
+- **The software is well-written.**
+  - Hint: no `.DS_Store` files visible in any folder of the repository
+  - Hint: the repository has a reasonable structure
+  - Hint: the software is written in functions
+  - Hint: there are no absolute paths in the software
+  - Hint: the software is packaged with or has documentation for obtaining demo data so you can quickly try to run the tool.
+- **Other people use the software.** This signals that other people believe in the method and that other people were able to install and use the software.
+  - Hint: the GitHub repository has a lot of stars
+  - Hint: the paper publishing the software has been cited frequently by other projects that used the tool.
+
+## Ideas for finding quality software
```

Comment:
Maybe also mention searching for repos directly on Github (may not be obvious to people that this is possible).